e2e/panels-suite/frontend-sandbox-panel.spec.ts

```diff
@@ -118,6 +109,13 @@ describe('Panel sandbox', () => {
       cy.wait(100); // small delay to prevent false positives from too fast tests
       cy.get('[data-sandbox-test="panel-editor"]').should('not.exist');
     });
+
+    it('Can access specific window global variables', () => {
+      cy.get('[data-testid="button-test-globals"]').click();
+      cy.get('[data-sandbox-global="Prism"]');
+      cy.get('[data-sandbox-global="jQuery"]');
+      cy.get('[data-sandbox-global="location"]');
```

Comment:
I'm curious what the error message would look like now that `cy.get` is not chained with an assertion? Maybe it would be more expressive with for example `cy.get(elem).should('be.visible')`

Reply:
The error will be a timeout not being to get the element. I can add `be.visible` check but It won't add to the validation itself.